### PR TITLE
[NPM] chore: NPM telemetry cleanup

### PR DESCRIPTION
**Pull request by JungukCho**
**Reason for Change**:

Remove dead codes in NPM-related metrics from telemetry and NPM

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Review by jaer-tsun — APPROVED** [review]
lgtm

File: npm/npm.go [hunk 1]
```diff
@@ -83,15 +81,10 @@ func NewNetworkPolicyManager(informerFactory informers.SharedInformerFactory, ex
...
-			PodCount:      0,
-			NsCount:       0,
-			NwPolicyCount: 0,
-		},
-		k8sServerVersion: k8sServerVersion,
-		NodeName:         os.Getenv("HOSTNAME"),
-		version:          npmVersion,
-		TelemetryEnabled: true,
+		k8sServerVersion:  k8sServerVersion,
+		NodeName:          os.Getenv("HOSTNAME"),
```
Comment (tamilmani1989) [inline npm/npm.go:85]: can you also add this(nodeName) for cni future PR if not present

**Review by tamilmani1989 — APPROVED** [review]
lgtm

File: npm/npm.go (hunk 1 above)
Comment (tamilmani1989) [reply npm/npm.go:85]: ignore.. AI wrapper package adds vm name

Comment (JungukCho) [reply npm/npm.go:85]: @tamilmani1989  I guess you meant in [CNIReport](https://github.com/Azure/azure-container-networking/blob/master/telemetry/telemetry.go#L79) struct?

Comment (tamilmani1989) [reply npm/npm.go:85]: ignore my comment.. nodename will get added by default in AItelemetry package

**Pull request merged**